modules/extension/mbstyle/src/main/java/org/geotools/mbstyle/parse/MBObjectParser.java

```diff
@@ -1426,9 +1440,74 @@ public Displacement displacement(JSONObject json, String tag, Displacement fallb
         }
     }
 
-    /** @return True if the layer has the provided property explicitly provided, False otherwise. */
+    //
+    // structure checks
+    //
+    /** @return True if json has the provided property explicitly provided, False otherwise. */
     public boolean isPropertyDefined(JSONObject json, String propertyName)
             throws MBFormatException {
+        return isDefined(json, propertyName);
+    }
```

Comment:
This doesn't add anything - why do both this method and `isDefined(JSONObject json, String propertyName)` exist (I see you've overloaded isDefined below, but why didn't you just overload this method instead)?

Reply:
I liked the way the `isDefined( json, propertyName)` method reads better, I should clean up references to this method.